test-cases/gutenberg/group.md

```diff
@@ -0,0 +1,333 @@
+-   Select `Group` block
+-   Check if none block gets dimmed
+
+Expected look:  
+![Dimming1](../resources/group-dimming-1.png)
+  
+![Dimming2](../resources/group-dimming-2.png)
+  
+![Dimming3](../resources/group-dimming-3.png)
+
+--------------------------------------------------------------------------------
+
+##### TC011
+
+### No extra margins applied when nesting empty group blocks
+
+-   Add new post
+-   Add a `Group` block
+-   Create some nested structure
+-   Select nested block to make some block gets dimmed
+-   Close post
+-   Reopen post
+-   Check if there is no block beeing selected and dimmed
+
+--------------------------------------------------------------------------------
+
+##### TC012
+
+### FloatingToolbar is render when group or it's child is selected
+
+-   Add a `Group` block
+-   Prepare nested structure
+-   Check if each time you select any block it renders `FloatingToolbar` above
+
+Expected look:  
+![FloatingToolbarRoot](../resources/group-floating-toolbar-root.png)
+  
+![FloatingToolbarNested](../resources/group-floating-toolbar-nested.png)
+
+--------------------------------------------------------------------------------
+
+##### TC013
+
+### Breadcrumbs on FloatingToolbar is properly displayed
+
+-   Add a `Group` block
+-   Create nested structure
+-   Check if each time you change selection `Breadcrumbs` show proper block title and icon 
+-   Check if each time you change selection `Breadcrumbs` show proper parent block
+-   Check if `Breadcrumbs` renders navigation up button
+
+Expected look:  
+![BreadcrumbsRoot](../resources/group-breadcrumbs-root.png)
+  
+![BreadcrumbsNested](../resources/group-breadcrumbs-nested.png)
+
+--------------------------------------------------------------------------------
+
+##### TC014
+
+### Navigation up works as expected
+
+-   Add a `Group` block
+-   Create nested structure
+-   Select deep nested block
+-   Press navigation up button on `Breadcrumb`
+-   Check if each time you press navigation up button the selection moves to parent block
+-   Check if border and dim styles changes accordingly
+
+--------------------------------------------------------------------------------
+
+##### TC015
+
+### Navigation down works as expected
+
+-   Add a `Group` block
+-   Create some nested structure ( at least 3 levels deep )
+-   Clear selection
+-   Press on the bottom-most block in hierarchy of added `Group`
+-   Check if the selection is redirect properly to top-most parent
+-   Repeat - press on the bottom-most block in hierarchy of added `Group`
+-   Check if each time you press nested block you move selection one level down (block which gets selection should be common ancestor with previously selected block and pressed block)
+
+--------------------------------------------------------------------------------
+
+##### TC016
+
+### Navigation in hierarchy works as expected
```

Comment:
OR:

```suggestion
### Cross navigation in between blocks works as expected
```

Reply:
Fixed, do you think we might need a GIF to present some behaviour of the navigation ?